python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -35,11 +36,10 @@ class LocalFile(IGangaFile):
                                      'subfiles': ComponentItem(category='gangafiles', defvalue=[], hidden=1,
                                                 sequence=1, copyable=0, doc="collected files from the wildcard namePattern"),
                                      'compressed': SimpleItem(defvalue=False, typelist=[bool], protected=0, doc='wheather the output file should be compressed before sending somewhere'),
-                                     #'output_location': SimpleItem(defvalue=None, typelist=[str, None], hidden=1, copyable=1, doc="path of output location on disk")
                                      })
     _category = 'gangafiles'
     _name = "LocalFile"
-    _exportmethods = ["location", "remove", "accessURL"]
+    _exportmethods = ["location", "remove", "accessURL", "copyTo"]
```

Comment:
It should be for others, in fact I'm sure I wrote a test for the `MassStorageFile` which tests this.
Each class decides what to expose itself, I'm not changing this as I'm not wanting to decide that in this PR.